test/assert.ts

```diff
@@ -14,6 +14,12 @@
  * limitations under the License.
  */
 
+declare namespace NodeJS {
+    interface Global {
+        assert: Chai.Assert;
```

Comment:
Confused why we need this and line 24. (I just don't get what's going on here, can you explain it?)